odml/tools/format_converter.py

```diff
@@ -176,9 +189,13 @@ def convert(cls, args=None):
         """
         parser = argparse.ArgumentParser(description="Convert directory with odml files to another format")
         parser.add_argument("input_dir", help="Path to input directory")
-        parser.add_argument("result_format", choices=['v1_1', 'odml'], help="Format of output files")
+        parser.add_argument("result_format", choices=['v1_1', 'odml', 'xml' 'pretty-xml', 'trix',
```

Comment:
Is it possible to use `._conversion_formats.keys()` for the `choices` argument as well? Then they are defined in exactly one place.